sha1-file.c

```diff
@@ -1416,7 +1417,7 @@ static int loose_object_info(struct repository *r,
 	return (status < 0) ? status : 0;
 }
 
-int fetch_if_missing = 1;
+repo.fetch_if_missing = 1;
```

Comment:
That is not the correct spot to initialize default values of structs. The correct spot is `repo_init()` in `repository.c`.